### PR TITLE
Move version checks from public container to static website

### PR DESCRIPTION
**Pull request by vibhansa-msft**
## ✅ What
 
Version check is done on a public container with anonymous access, but due to security issues the public access will be terminated soon.
Now onwards the version check with work through a static website hosted on same storage account,
 
## 🤔 Why
 
Security vulnerability restricting anonymous access.
 
## 👩‍🔬 How to validate if applicable
 
Check the URI used to retrieve the latest version
 
## 🔖 Related links
NA

### CODE REVIEW

**Comment by vibhansa-msft** [comment]
Current changeset is getting an index.xml file from the website and expects it will have latest version written under \<latest\> tag in that xml file.

File: cmd/root.go [hunk 1]
```diff
@@ -104,18 +103,23 @@ func getRemoteVersion(req string) (string, error) {
 		return "", err
 	}
 
+	if len(body) > 50 {
```
Comment (souravgupta-msft) [inline cmd/root.go:106]: Should we do something like pattern based here instead of length?

Comment (vibhansa-msft) [reply cmd/root.go:106]: After the check we are doing unmarshall of the xml contents so pattern matching will fail if the expected tags are not present. Hence did not do that explicitly.

File: common/version.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ import (
 	"strings"
 )
 
-const Blobfuse2ListContainerURL = "https://blobfuse2.blob.core.windows.net/release"
+const Blobfuse2ListContainerURL = "https://blobfuse2.z13.web.core.windows.net/release"
```
Comment (souravgupta-msft) [inline common/version.go:42]: Can we make a short URL for this. Also please add a comment what this short URL will have for reference.

File: blobfuse2-release.yaml [hunk 3]
```diff
@@ -1782,7 +1797,39 @@ stages:
                 workingDirectory: $(root_dir)
 
               - script: |
-                  python3 releaseVersionUpdate.py "$(VERSION_CNT_SAS_URL)" "`blobfuse2 version`"
```
Comment (souravgupta-msft) [inline blobfuse2-release.yaml:1785]: This python script can be removed as it is not being used anywhere else

**Review by souravgupta-msft — APPROVED** [review]

File: common/version.go (hunk 2 above)
Comment (vibhansa-msft) [reply common/version.go:42]: does not work as the aka.ms link needs exact url destination and we suffix the url with the version numbers at times.

**Review by ashruti-msft — APPROVED** [review]

**Review by souravgupta-msft — APPROVED** [review]

**Pull request merged**